nss-day-cohort-41/tabloidcli-fans-of-fozzie-mo
# Sf refactor

# Description
1. Created conditional for if user inputs more than 55 characters for title (post, journal, blog);
2. Created try/catch for if date does not meet criteria for SQL when adding a post
3. Created conditional for user input in edit blog, if more than 55 characters(post, journal, blog);
## Type of change
Please delete options that are not relevant.
# Testing Instructions
pull down sf-refactor branch; 
1. select post management from main menu
2. select add post
3. input information into fields
4. input date that is less than 1753
5. select author and blog
6. should get error message for invalid date response and bring you back to post menu;
7. repeat for edit post

For blog and journal, select respective menu options to add a blog and journal and attempt to skip over the title fields and then attempt to input a string longer than 55 characters- you should get a warning saying you can't do this and will be re-prompted to try again

Repeat for edit blog and journal;

# Checklist:
